rslaser/elements/crystalSlice.py

```diff
@@ -1,26 +1,29 @@
 import math
 import numpy as np
-import srwlib 
+import srwlib
 from array import array
 from pykern.pkcollections import PKDict
 
 import rslaser.rsoptics as rso
 #from rslaser.rsoptics.element import *
 
-c_light = 299792458.0  # m/s, in vacuum 
+c_light = 299792458.0  # m/s, in vacuum
```

Comment:
scipy.constants.c

Reply:
I agree that c should be taken from scipy.constants